doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -712,9 +712,9 @@ And this maximum is:
 
 If this is not clear enough you can take a look at the edge labels in the above graph.
 
-To choose a base for a tree we need to apply the previous algorithm to all the roots of tree.
+To choose a base for a tree we, need to apply the previous algorithm to all the roots of tree.
```

Comment:
We're talking about one tree but I realized that you could have sthg like : 
A   B   C
  \  /  \  /
  x1  x2
I never saw it happen, but I couldn't prove it wouldn't... And I didn't want to change the 'tree' name everywhere -_-